src/scripts/main.js

```diff
@@ -1 +1,65 @@
 'use strict';
+
+let right;
+let left;
+
+function display(order) {
+  const box1 = document.createElement('div');
+
+  document.body.appendChild(box1);
+  box1.textContent = `${order}`;
+
+  order.split(' ').includes('resolved')
+    ? box1.className += 'succes'
+    : box1.className += 'warning';
```

Comment:
```suggestion
  order.split(' ').includes('resolved')
    ? box1.className += 'success'
    : box1.className += 'warning';
```
You can also pass the second argument to function with success/warning to make it easier